src/main/java/com/teamrocket/tms/utils/calculators/CompletionCalculator.java

```diff
@@ -0,0 +1,29 @@
+package com.teamrocket.tms.utils.calculators;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.List;
+import java.util.Map;
+
+public class CompletionCalculator {
+    public static double getPercentageComplete(Map itemsMap){
+        List<Boolean> valueList = itemsMap.values().stream().toList();
+        int completedObjectives = (int)valueList.stream()
+                .filter(item -> item)
+                .count();
+        double result = (double) completedObjectives / itemsMap.size() * 100;
+
+        return roundUp(result, 2);
```

Comment:
Can we maybe use a constant for 2, and use a description that can let all team members know (or anyone is looking at the code) why we used them? Maybe even a constant like `TWO_DECIMALS` can help.

Usually we call those numbers `magic numbers`. We don't want to use magic numbers in our code.

Reply:
thanks for the feedback, totally right!